src/modules/Events/constants.ts

```diff
@@ -0,0 +1,7 @@
+import { Event } from 'src/core/api';
+
+export const generateEventsId = (events: Event[]) => {
+  return {
+    tech4impact: events.find(event => event.title === 'Tech for Impact Summit 2024')?.id,
```

Comment:
I don't get why we use this function? :D
Why a simple map to id cannot do the work?